src/main/java/frc/robot/robot_manager/RobotCommands.java

```diff
@@ -104,4 +106,14 @@ public Command unjamCommand() {
     return Commands.runOnce(robot::unjamRequest, requirements)
         .andThen(robot.waitForState(RobotState.IDLE_NO_GP));
   }
+
+  public Command speakerOrAmpCommand() {
```

Comment:
this function shouldn't exist, we have this logic in the confirmShot request already. we can tweak that as needed